example/precheck

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# Minimum versions
+MIN_DOCKER_VERSION="22.0"
```

Comment:
Do you remember why we require docker 22? I am on 20 and it works™

Reply:
Never mind. The `flake.nix` still installed 20.10 because I never updated it. 20.10 is end of life, so not going down to that version sounds reasonable.

Reply:
BTW, I was running docker daemon 24 with docker client 20, the script only considered the client version. Might be good enough for now, but wanted to flag it.